odtp/dashboard/utils/ui_theme.py

```diff
@@ -34,6 +33,7 @@ def frame(navtitle: str):
     )
     with ui.header().classes("justify-between text-white"):
         ui.label("OTDP").classes("font-bold")
+        ui.link("Log out", PATH_SIGN).classes("text-left",replace="text-white")
```

Comment:
todo: change `PATH_SIGN` to something else as this is a `logout`: the variable name is confusing here. Also on logout the storage needs to be cleared. After logout, it still showed me as the dashboard.